front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -48,9 +48,9 @@ const StyledIdeasShow = styled(IdeasShow)`
   padding-left: 60px;
   padding-right: 60px;
 
-  ${media.tablet`
-    min-height: calc(100vh - ${({ theme: { mobileTopBarHeight } }) =>
-      mobileTopBarHeight}px);
+  ${({ theme }) => media.tablet`
+    margin-top: ${theme.menuHeight}px;
```

Comment:
Page was suddenly hiding under the nav on mobile. Couldn't figure out how that was happening but fixed :)